### PR TITLE
Update UPGRADING for 4.0

### PR DESCRIPTION
**Pull request by bernd**
Add more upgrading notes.

### CODE REVIEW

File: UPGRADING.rst [hunk 1]
```diff
@@ -4,79 +4,156 @@ Upgrading to Graylog 4.0.x
...
+Changes to the Elasticsearch Support
+------------------------------------
+
+Starting with Graylog v4.0, bigger changes to the Elasticsearch versions supported are happening:
+
+  - Support for Elasticsearch versions prior to v6.8.0 are dropped.
+  - Support for Elasticsearch v7.x is now included.
+
+This means that you can upgrade to Graylog v4.0 without an Elasticsearch update only if you have been on at least Elasticsearch v6.8.0 before.
+Additionally, due to the fact that Elasticsearch supports only indices created by the last two major versions (i.e. ES6.8.0+ reads indices created by ES5 & ES6, ES7 reads indices created by ES6 & ES7), you can change to Graylog v4.0 with an Elasticsearch update without reindexing only if you have been on at least Elasticsearch v6.0.0 before.
```
Comment (mpfz0r) [inline UPGRADING.rst:23]:
```suggestion
Additionally, due to the fact that Elasticsearch supports only indices created by the last two major versions (i.e. ES6.8.0+ reads indices created by ES5 & ES6, while ES7 reads indices created by ES6 & ES7), you can change to Graylog v4.0 with an Elasticsearch update without reindexing only if you have been on at least Elasticsearch v6.0.0 before.
```
I find it easier to read that way

File: UPGRADING.rst [hunk 2]
```diff
@@ -4,79 +4,156 @@ Upgrading to Graylog 4.0.x
...
+=========================
+
+The following API endpoints are deprecated beginning with 4.0.
+
+Deprecation of cluster stats endpoints
+--------------------------------------
+
+Starting with v4.0, the cluster stats endpoints are deprecated and will be removed in a future version. Those include:
+
+- ``/system/cluster/stats`
```
Comment (mpfz0r) [inline UPGRADING.rst:93]:
```suggestion
- ``/system/cluster/stats``
```
I guess that needs two backticks as well

**Review by mpfz0r — APPROVED** [review]

File: UPGRADING.rst [hunk 3]
```diff
@@ -4,79 +4,156 @@ Upgrading to Graylog 4.0.x
...
-Starting with v4.0, the cluster stats endpoints are deprecated and will be removed in a future version. Those include:
+Breaking Changes
+================
+
+Changes to the Elasticsearch Support
+------------------------------------
+
+Starting with Graylog v4.0, bigger changes to the Elasticsearch versions supported are happening:
+
+  - Support for Elasticsearch versions prior to v6.8.0 are dropped.
```
Comment (dennisoelkers) [inline UPGRADING.rst:19]: I know I am correcting myself, but it should be `is dropped`, not `are dropped` :)

Comment (bernd) [reply UPGRADING.rst]: @dennisoelkers Thanks, I pushed a fix!

**Pull request merged**